pwm_frequency: 47.2
pwm_ms_bias: 0.055

thruster_ch_list: [0, 3, 4, 5]
thruster_topic_list: [thruster_1, thruster_2, thruster_3, thruster_4]
thruster_min_us: [1100, 1100, 1100, 1100]
thruster_max_us: [1900, 1900, 1900, 1900]
thruster_init_us: [1500, 1500, 1500, 1500]
thruster_direction: [1, 1, 1, 1]  #1 or -1

# led_ch_list: [5]
# led_topic_list: [led_1]
# led_min_us: [1100]
# led_max_us: [1900]
# led_init_us: [1100]

# servo_ch_list: [6]
# servo_topic_list: [servo_1]
# servo_min_us: [300]
# servo_max_us: [2600]
# servo_center_us: [1500]